lib/install/validate/validator.go

```diff
@@ -368,6 +427,13 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *config.
 	// port forwarding
 	conf.AddNetwork(bridgeNet)
 
+	if v.IsVC() {
+		err := v.checkVDSMembership(ctx, input.BridgeNetworkName)
```

Comment:
Cause your method is always checked after the network validation is done, so suppose you could pass in the dpg reference directly into your function.